Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -92,11 +87,29 @@ class TextureCacheBase
       other_entry->references.emplace(this);
     }
 
+    enum FlagPos
+    {
+      EFB_COPY_FLAG_POS = 0,
```

Comment:
I think you might be better off using a [Bit field](http://en.cppreference.com/w/cpp/language/bit_field) rather than a bitset and this enum.

With a Bit field, it would be stupidly easy and sane to put the flags and the format into a single u32.